### ISSUE TITLE
New Blender Character Modelling Campaign (Little Mike Gone)

### ISSUE THREAD

**Issue by fjbagley** [timberline-secondary/hackerspace-todo#188]
Little Mike is now private on YouTube. That was an old campaign anyway (Blender 2.7). That leaves Lego character which is also 2.7.

I have two students testing this 9 part modelling and rigging series:

https://www.youtube.com/watch?v=UKI8_PAFFz4&list=PLjEOB0vlBkKhw_xIG4FQZWc0BJR7idR9y

I'm also looking for other options.

**Comment by fjbagley** [comment]
A significant part of the campaign is sculpting. The students that have tried it for me have struggled with this aspect. 

Almost all of my current students are Grade 9s as there is a different schedule this year, but I will try this one next:
[https://www.youtube.com/watch?v=E5HnnGcGvmc&list=PLn3ukorJv4vuOi2Ar-Xt46VFbDFP3c_RE&index=2&ab_channel=GrantAbbitt](url)

**Comment by tylerecouture** [comment]
I like Grant Abbit!  Also he does a quick rig and animation at the end which is cool.

**Comment by fjbagley** [comment]
This is 2.8 from earlier this year and we blender 3.0 is no longer beta. Apparently they are increasing the version number at a faster rate (not necessarily as much changed in the software). The Lego man which this might replace was 2.7, but 6 years ago.

From my reading on Blender it seems they will stop moving things around as much in future versions.

I have nobody even close to this stage this rotation. I will hold off on this and see if Grant Abbott makes a version 3 tutorial in the next month.

**Comment by tylerecouture** [comment]
I am leaving the Lego Man in, even though it's 2.7 I've had many students go through it in 2.8 with almost no issues, other than they get some practice figuring out how to find features that have moved =)

All the specific topics and modelling methods are still relevant!

**Comment by fjbagley** [comment]
Yes, good point. In some ways just copying a video takes far less skill
than understanding where buttons should be or how to work around them. I
still hope a 3.0 make a character and rig it tutorial series will show up,
so I will leave this open for now.

**Comment by tylerecouture** [comment]
#116
